.github/workflows/ci.yaml

```diff
@@ -158,7 +158,7 @@ jobs:
     steps:
       - id: release-branch
         run: |
-          echo "branch=$([[ "${{ github.head_ref || github.ref_name }}" =~ "^[0-9]+\.[0-9]+\.[0-9]+$" ]]  && echo "true" || echo "false" )" >> "$GITHUB_OUTPUT"
```

Comment:
What's the need for this change?

Reply:
unfortunately cant have quotes on regex, it doesnt work found while testing on a pr opened on my fork